collections/eton college windsor/Eton_450.xml

```diff
@@ -16,7 +16,7 @@
 				<address>
 					<addrLine>Eton College</addrLine>
 					<addrLine>College Library</addrLine>
-					<street/>
+					<street></street>
```

Comment:
If `<street>` is not a required tag it should probably just be removed, rather than left empty.